HikVisionSource/README.md

```diff
@@ -0,0 +1,217 @@
+# Overview
+
+The following documentation outlines how to incorporate a HikVision source as part of your project.
+This allows a user to construct applications that integrate with a HikVision camera and are able to detect notifications which contains images and information about face, movement, and heat.
+The HikExtension source enables sending zoom and movement commands to the camera along with support for other scenarios such as monitoring applications. etc.
+The extension creates, as part of handling incoming notification from the camera,
+files in a defined folder and the uploading of the file content as events to Vantiq.
+The extension source enables control on the names of the different attributes based on their order in the HikVision file and changing the name of or deleting after processing the file.
+The extension source can handle multiple files in parallel;
+this can be controlled by parameters outlined in the config section.
+The extension source can support mutiple cameras in parallel, based on configuration.  
+
+In order to incorporate this Extension Source, you will need to set up your local machine with a client JRE that can connect to your HikVision camera.
+Once you have done this, you will need to create the Source in Vantiq.
+The documentation
+has been split into two parts, [Setting Up Your Machine](#machine) and [Setting Up Your VANTIQ](#vantiq).
+
+# Prerequisites <a name="pre" id="pre"></a>
+
```

Comment:
We should probably state here that this source can only be built or operated on linux or windows machines (include versions if applicable).  Specifically, there is no Macintosh support (that I know of).  (Is that correct?)

Reply:
I asked Oliveir help in order to add the Linux part . ( this are the two additional pools ... ) 
I can manual merge his insight to that pool request if you want. 
I don't think anyone tried to run it on mac yet...
would you consider review Olivier addition s , or first i should merge it ?

Reply:
He proposed that into master, I think, but that's not really relevant.  He also proposed a PR into your branch.

I suggested that you two agree on how to do it but put it into this branch.  He can just pull your branch, make the changes, commit & push, then you re-pull (just like you & I have done a few times).  I don't need to (nor want to) have to approve that as a PR.

Reply:
ok